### PR TITLE
ISSUE-11 Introduce method to update a User.

### PR DESCRIPTION
**Pull request by martijnhaex**
Closing #11

### CODE REVIEW

File: src/main/java/be/jidoka/jdk/keycloak/admin/domain/HasUserPersonalData.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package be.jidoka.jdk.keycloak.admin.domain;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+public interface HasUserPersonalData {
```
Comment (bastijns-jeroen) [inline src/main/java/be/jidoka/jdk/keycloak/admin/domain/HasUserPersonalData.java:7]: UpdateUserPersonalData?

File: src/test/java/be/jidoka/jdk/keycloak/admin/service/KeycloakUserAdminServiceUpdateUserTest.java [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+	@Autowired
+	private UsersResource usersResource;
+
+	@Test
+	void updatesAttributesAndRequiredActionsOnly() {
+		CreateUser createUser = bertenBoedhoe();
+		String userId = keycloakUserAdminService.createUser(createUser);
+
+		UpdateUserBuilder updateUser = UpdateUserBuilder.builder()
+				.userId(userId)
```
Comment (bastijns-jeroen) [inline src/test/java/be/jidoka/jdk/keycloak/admin/service/KeycloakUserAdminServiceUpdateUserTest.java:36]: The test claims to update something, but I don't see it here

File: src/main/java/be/jidoka/jdk/keycloak/admin/domain/HasUserPersonalData.java (hunk 1 above)
Comment (martijnhaex) [reply src/main/java/be/jidoka/jdk/keycloak/admin/domain/HasUserPersonalData.java:7]: Done updated to UserPersonalDataCommand.

File: src/test/java/be/jidoka/jdk/keycloak/admin/service/KeycloakUserAdminServiceUpdateUserTest.java (hunk 2 above)
Comment (martijnhaex) [reply src/test/java/be/jidoka/jdk/keycloak/admin/service/KeycloakUserAdminServiceUpdateUserTest.java:36]: Renamed, extracted some basic assertions to a specific Assertion utility class.

**Pull request merged**